ruby/ruby_trunk/ruby_benchmarks/.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - sudo apt-get install -y curl
+  - sudo apt-get install -y jq
+
+script:
+  - sed -i "s_FROM.*_FROM rubybench/ruby_trunk:$(curl -s -S "https://registry.hub.docker.com/v2/repositories/rubybench/ruby_trunk/tags/" | jq '."results"[]["name"]'| sed -n 1p |cut -c2-8)_" ./Dockerfile
```

Comment:
What we want here is to build a ruby base image first that is not tied to any particular benchmark. 

`rubybench/ruby:c25fa0fa9e463d33854d1b0c421614ca34890e98` will point to https://github.com/ruby/ruby/commit/c25fa0fa9e463d33854d1b0c421614ca34890e98. Once we have that, we can build `rubybench/ruby_trunk` using a `FROM rubybench/ruby:c25fa0fa9e463d33854d1b0c421614ca34890e98`